lib/Core/Site/Values/Content.php

```diff
@@ -143,4 +176,42 @@ private function buildField(array $properties = [])
         $this->fields[$field->fieldDefIdentifier] = $field;
         $this->fieldsById[$field->id] = $field;
     }
+
+    private function getLocations()
```

Comment:
Added in https://github.com/netgen/ezplatform-site-api/commit/654be5ffd01d4f1f4f74246020e2e9e899a4e7f4

Any preference on how these should be sorted? It should be done at least for consistence. I was thinking depth then path. Priority is another option, but I think it makes sense only when used among siblings.